sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/VisualStudioCacheAccessor.java

```diff
@@ -131,7 +131,7 @@ public String getCredentials(String serviceName, String accountName) {
 
                 byte[] readCreds = keyRingAccessor.read();
                 credential = new String(readCreds, StandardCharsets.UTF_8);
-            } catch (Exception e) {
+            } catch (Exception | UnsatisfiedLinkError e) {
```

Comment:
Why not use `Throwable`?

Reply:
Throwable will work too, but didn't want to aggressively block other Error types which may qualify to fail the credential.